mw_jwt.go

```diff
@@ -424,7 +463,7 @@ func (k *JWTMiddleware) ProcessRequest(w http.ResponseWriter, r *http.Request, _
 		if k.Spec.JWTSigningMethod == "rsa" {
 			asRSA, err := jwt.ParseRSAPublicKeyFromPEM(val)
 			if err != nil {
-				log.Error("Failed to deccode JWT to RSA type")
+				log.Error("Failed to decode JWT to RSA type")
```

Comment:
maybe also log error here: `log.WithError(err).Error("Failed to decode JWT to RSA type")` so we have more context in log

Reply:
ok